src/main/scala/com/fulcrumgenomics/util/SimpleCounter.scala

```diff
@@ -39,17 +37,23 @@ object SimpleCounter {
   * zero for any item that has not been counted yet.  
   */
 class SimpleCounter[T] extends Iterable[(T, Long)] {
-  private val counts = mutable.Map[T,Long]()
+  import scala.collection.JavaConversions._
+
+  private val counts = new java.util.TreeMap[T,Long]()
```

Comment:
What happened to testing to see if `T` was `Comparable`?  This will now fail if given a non-`Comparable` type T.